### ISSUE TITLE
Schneider Wiser with Homey

### ISSUE THREAD

**Issue by Norinvstor** [Signalare/com.se.wiserapp#54]
Hi,

If there already is information about this, please enlighten me and delete my post.

Do you need the Wiser gateway to for example connect the Wiser WDE002388 (dimmer)(Zigbee) to Homey?
Is there any official thread for this topic in the Homey forum?

Regards

**Comment by Norinvstor** [comment]
Thank you for the fast reply! I have asked in the Homey Community forum and sent an email to Schneider aswell.
If I buy it I might not be able to return it.

Is it you that made the app in Homey app store?

**Comment by h3rman** [comment]
Yes it is. Just so you know WDE002388 is discontinued. I think it's replaced by WDE002961 that should handle small LED loads better and where the neutral is optional.

**Comment by h3rman** [comment]
![WDE002388](https://user-images.githubusercontent.com/1621579/191084163-20332dc4-efcf-412d-8b4c-0698eab1fe75.png)
[Product page (en)](https://www.se.com/ww/en/product/WDE002388)
[Instruction sheet WDE00x388_HW.pdf](https://github.com/Signalare/com.se.wiserapp/files/9601271/WDE00x388_HW.pdf)

**Comment by Norinvstor** [comment]
Thank you! Can you find any Norwegian side for this replacement dimmer?

**Comment by h3rman** [comment]
don't know
